doc-content/enterprise-only/project-data/data-objects-create-proc.adoc

```diff
@@ -12,12 +12,12 @@ The following procedure is a generic overview of creating data objects. It is no
 .Procedure
 . In {CENTRAL}, go to *Menu* -> *Design* -> *Projects* and click the project name.
 . Click *Add Asset* → *Data Object*.
-. Enter a unique *Data Object* name and select the *Package* where you want the data object to be available for other rule assets. Data objects with the same name cannot exist in the same package. The package that you specify must be the same package where the rule assets that require those data objects have been assigned or will be assigned.
+. Enter a unique *Data Object* name and select the *Package* where you want the data object to be available for other rule assets. Data objects with the same name cannot exist in the same package. In the DRL file you specify, you can import a data object from any package.
```

Comment:
"In the DRL file you specify, you can import a data object from any package."
Suggestions to avoid double verb sentence:
"In the specified DRL file,...."
"In your DRL file,..."

Reply:
Changed as suggested to "In the specified DRL file,....".